provisioning/miniprov/hare_mp/main.py

```diff
@@ -0,0 +1,328 @@
+import argparse
```

Comment:
is this new hare_setup.py? the code is duplicated from provisioning/hare_setup.py.

Reply:
Yes, it is duplicated. I just didn't remove the old hare_setup.py (was afraid to break Provisioning if they rely on that script)